tests/rt.sh

```diff
@@ -739,6 +739,40 @@ case ${MACHINE_ID} in
     STMP=${STMP:-${dprefix}/RT_BASELINE}
     PTMP=${PTMP:-${dprefix}/RT_RUNDIRS} 
 
+    SCHEDULER="slurm"
```

Comment:
You can replace these couple of lines with:
```
  gaeac6)
    echo "rt.sh: Setting up gaea c6..."
    if [[ "${ROCOTO:-false}" == true ]] ; then
      module use /ncrc/proj/epic/c6/modulefiles
      module load rocoto/1.3.7
      ROCOTO_SCHEDULER="slurm"
    fi

    export LD_PRELOAD=/usr/lib64/libstdc++.so.6
    module use /ncrc/proj/epic/spack-stack/c6/spack-stack-1.6.0/envs/fms-2024.01/install/modulefiles/Core
    module load stack-intel/2023.2.0
    module load cray-mpich/8.1.29
    module load python/3.10.13
#RV do we still need this:
#   module load gcc-native/12.3
    if [[ "${ECFLOW:-false}" == true ]] ; then
      module use /ncrc/proj/epic/spack-stack/modulefiles
      module load ecflow/5.8.4
      ECF_HOST=$(hostname)
      ECF_PORT=$(( $(id -u) + 1500 ))
      export ECF_PORT ECF_HOST
    fi
```
Natalie installed rocoto on C6 (included in code change suggestion).
When ecflow is done we can replace that one as well, but for now, installation from C5 is working fine on C6.
Question is do we need to load gcc anymore?